tests/Scheduling/ScheduleListCommandTest.php

```
@@ -0,0 +1,135 @@
+    protected $signature = 'foo:command';
+
+    protected $description = 'This is the description of the command.';
+}
+
+class FooJob
+{
+}
+
+class FooParamJob
+{
+    public function __construct($param)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$param'. (undefined)

(UnusedFormalParameter)